### PR TITLE
MTP-1453 my list page

### PR DESCRIPTION
**Pull request by Staberinde**
https://dsdmoj.atlassian.net/browse/MTP-1453

### CODE REVIEW

File: .floo [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
[the remark this answers is not in the document]
Comment (Staberinde) [reply .floo:1]: I added those on purpose, essentially Floo is a remote pair programming framework which myself and Matt tried out. Seems to work quite well actually. Floo has the concept of a project, which is essentially a directory space synced between the developers working on a ticket together; this is just a metadata file associated with that. Happy to remove it if you want though

[the remark this answers is not in the document]
Comment (mattbee) [reply .floo:1]: Or we could split it into a separate commit detailing what the files are and that they can be ignored (easier to then revert that commit in the future)

[the remark this answers is not in the document]
Comment (ushkarev) [reply .floo:1]: i haven't used it but i'm sure it's a useful tool. i just feel that it's not part of the project code. it allows someone to remotely edit files on your machine, doesn't it? i don't think that should be publicly listed here

[the remark this answers is not in the document]
Comment (ushkarev) [reply .floo:1]: i was just able to edit a file in incognito… this feels like an extremely dangerous thing to publicly post on the internet

**Pull request merged**